.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+*Making a contribution to the code base?* Please read [this](https://github.com/sinonjs/sinon/blob/master/CONTRIBUTING.md#contributing-to-the-code-base).
```

Comment:
This file repeats a lot of what is said in `CONTRIBUTING.md`. I think it would be more valuable, if instead we could make it easy to copy-paste into a PR and fill in the sections

```markdown
#### Purpose (TL;DR) - mandatory
> give a concise (one or two short sentences) description of what what problem is being solved by this PR
>
> Example: Fix issue #123456 by re-structuring the colour selection conditional in method `paintBlue`

#### Background (Problem in detail)  - optional
> When relevant, give a more thorough description of what the problem the PR is trying to solve. Examples of good topics for this section are:
> * Link to an existing GitHub issue describing the problem
> * Describing the problem in greater detail than the TL;DR section above
> * How you discovered the issue, if it's not already described as an issue on GitHub
> * Discussion of different approaches to solving this problem and why you chose your proposed solution

#### Solution  - optional
> When contributing code (and not just fixing typos, documentation and configuration), please describe why/how your solution works. This helps reviewers spot any mistakes in the implementation.
>
> Example: 
> "This solution works by adding a `paintBlue()` method"
> Then your reviewer might spot a mistake in the implementation, if `paintBlue()` uses the colour red.

#### How to verify - mandatory
1. Check out this branch (see github instructions below)
2. `npm install`
3. <your-steps-here>
```